packages/apps/docs/package.json

```diff
@@ -24,10 +24,12 @@
     "cypress:run": "percy exec -- cypress run",
     "dev": "pnpm run build:scripts && next dev",
     "format": "pnpm run /^format:.*/",
+    "format:lint": "pnpm run lint:src --fix",
     "format:src": "prettier . --cache --write",
     "lint": "pnpm run /^lint:.*/",
+    "lint:fmt": "prettier . --cache --check",
```

Comment:
Where does `fmt` stand for btw?

Reply:
format (see e.g. https://dprint.dev/cli/#formatting-files)

Reply:
Right, so we got `format:lint` and `lint:format`, gotcha

Reply:
lol, that's... odd/twisted

Ideas welcome! In practice it's "auto-save" in IDE and `pnpm lint` or `pnpm format` in the shell, so not that problematic I guess.

Reply:
Wasn't meant sarcastically btw, just verifying :D Thanks for the clarification!

Reply:
Not taken that way! I didn't see it myself, and it is odd. But not wrong :)

Reply:
Get a room you guys.